__DEFINES/global.dm

```diff
@@ -367,6 +367,15 @@ var/list/blacklisted_mobs = list(
 		/mob/living/adamantine_dust // Ditto
 		)
 
+//Boss monster list
+var/list/boss_mobs = existing_typesof(
```

Comment:
Does this work if it hasn't been added to globalaccess?

Reply:
It will work but you won't be able access it from SDQL and similar.

Reply:
`globalaccess` is obsolete as of 512 once someone gets around to changing how SDQL accesses global vars anyway